extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -276,6 +276,10 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 	}
 
+	async removeNotebook(bookItem: BookTreeItem): Promise<void> {
+		await this.bookTocManager.removeNotebook(bookItem);
```

Comment:
You can just return the call directly - no need to await here (that's just unnecessary overhead)